modules/admin_manual/examples/installation/post-installation-steps.sh

```diff
@@ -37,22 +37,42 @@ rootuser='root'
 
 # Because the data directory can be huge or on external storage, an automatic chmod/chown can take a while.
 # Therefore this directory can be treated differently.
-# If you have already created an external data and apps-external directory which you want to link,
+# If you have already created an external "data" and "apps-external" directory which you want to link,
 # set the paths above accordingly. This script can link and set the proper rights and permissions
 # depending what you enter when running the script.
-# You have to run this script twice, one time to prepare installation and one time post installation
 
-# In case you upgrade an existing installation, your original directory will be renamed including a timestamp 
+# When the instance is setup, either post a fresh install or, after an upgrade, run this script again - but
+# only for securing ".htaccess files". This sets the appropriate ownership and permissions for them.
 
-# Example input
-# New install using mkdir:     n/n/n/n (create missing directories, setup permissions and ownership)
+# In case you upgrade an existing installation, your original directory will be renamed, including a timestamp.
```

Comment:
This was existing text, so I will just comment for the future that this sort of use of "In case" sounds unusual. "In case you..." is better and more simply written "If you..."

Reply:
Thanks, will keep it in mind 👍